index.md

```diff
@@ -1,25 +1,224 @@
+
+### Figma
+
+Our designers work with **Figma**. You can download it here:
+
+* [Figma](https://www.figma.com/downloads/)
+
+To access the designs you need to log in with your Wisemen account:
+
+* [Figma wireframes](https://www.figma.com/file/hebgv4Qx8VanMAQkO1NFpa/Onboarding-to-do?node-id=407-4095&t=2qdyy89lKwN7dFw3-0)
+
+### BitBucket repository
+
+*ToDo: Add link to BitBucket repository*
+
+### Jira access
+
+*ToDo: Add link to Jira*
+
+## Create the project
+
+Duration: 0:50:00
+
+### 3.1 Create a new project
+
+Open Android Studio and create a new project. You can name it Wiselab_Android_YOURNAME.
+
+![](index/img/setup/new_project.png)
+
+### 3.2 Link your project to BitBucket
+
+You can find the link to your repo on your [personal Confluence](https://appwise.atlassian.net/wiki/home) page.
+
+![](index/img/setup/bitbucket_clone.png)
+
+#### Use the terminal or IDE to link your project to BitBucket
+
+We will show you how to work with the ***terminal***. If you prefer to work with the ***IDE***, you can find the
+instructions [here](https://www.jetbrains.com/help/idea/set-up-a-git-repository.html#add-remote).
+
+* Open the terminal in Android Studio (bottom left corner)
+* add remote origin to project
+* ```shell
+  git init
+  git remote add origin <your repo url>
+  ```
+
+From here on you can choose to use the terminal or the IDE to work with Git.
+
+### 3.3 Our Branching strategy
+
+We use the **Gitflow** branching strategy. You can find more information about this
+strategy [here](https://www.atlassian.com/git/tutorials/comparing-workflows/gitflow-workflow).
+
+* **master** branch: this branch contains the latest release of the app
+* **staging** branch: this branch is used send to the client for testing
+* **QA** branch: this branch is used to test the features before they are merged into the develop branch
+* **develop** branch: this branch contains the latest development changes
+* **feature/...** branches: these branches are used to develop new features for the upcoming release
+* **bugfix/...** branches: these branches are used to fix bugs in the app
+
+Make sure you have created all 4 'standard' branches. You can do this by either using the terminal, IDE or a GUI
+tool like [SourceTree](https://www.sourcetreeapp.com/) or [Fork](https://git-fork.com/).
+
+Now checkout the **develop** branch and create a new feature branch called **feature/setup-project**.
+
+## Add the Android Core Library
+
+Duration: 0:40:00
+
+Now that you have created your project, it's time to add
+our [Android Core Library](https://github.com/appwise-labs/AndroidCore)! This library contains a lot of useful
+classes and functions that we use in all our projects. The latest version is also noted in the readme.
+
+### 4.1 Open the **build.gradle** file of your **project**
+
+* Add the following **buildscript** to the top of the file:
+
+```gradle
+buildscript {
+    ext {
+        android_core_version = "1.4.3"
```

Comment:
I might be better to keep these versions abstract e.g.: "<latest version>" as these versions change quite frequently and having them hardcoded here might result in starters copy-pasting them without checking if there is a newer version of a library